Docs: Troubleshooting updates.

jrajahalme opened the pull request:
Minor fixes to the troubleshooting docs.

Signed-off-by: Jarno Rajahalme <jarno@covalent.io>

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cilium/cilium/4569)
<!-- Reviewable:end -->

raybejjani commented on Documentation/troubleshooting.rst:
```diff
@@ -88,7 +89,8 @@ of all nodes in the cluster:
 
-To retrieve log files of a cilium pod, run
+To retrieve log files of a cilium pod, run (replace `cilium-1234` with a pod
+name returned by `kubectl -n kube-system get pods -l k8s-app=cilium`)
```
Would making the code snippet:
```
$ CILIUM_POD=$(kubectl -n kube-system get pods -l k8s-app=cilium)
$ kubectl -n kube-system logs --timestamps $CILIUM_POD
```
simplify this? With the added parenthetical the user now has to think about what these commands do but they may not fully understand. I'm assuming the worst-case panic for the troubleshooting, so the simpler the instructions the more likely it will be effective.
It could also be a one-liner, but I'm not sure how well it would format.

aanm replied:
`kubectl -n kube-system get pods -l k8s-app=cilium` might give you more than one pod

raybejjani replied:
hmm, true... maybe it isn't simpler then :/

raybejjani commented on Documentation/troubleshooting.rst:
```diff
@@ -166,12 +171,12 @@ minute.
+the networking level. The tool
 ``cilium monitor`` allows you to quickly inspect and see if and where packet
-drops happen.
+drops happen. Following is an example output (use `kubectl` as in previous
```
Could you switch this to say "use `kubectl exec`" so it's very clear which aspect of kubectl we want to use.
Similar to the other comment, it might also be worth explaining how to get the correct pod. Since we're providing instructions for both k8s and non-k8s, it might be easiest to use tabs with different snippets in each.

jrajahalme replied:
Thanks for the review!

Changed `kubectl` to `kubectl exec`.

jrajahalme committed:
Docs: Remove CoreOS Installation Guide

CoreOS links we use are now redirected to Tectonic installation guides. As such this document would require extensive revision and validation work. This has been broken for a while now and there has been no apparent interest in this guide, so better remove it.

Besides, users can use Kops instead, as it also installs CoreOS images.

Signed-off-by: Jarno Rajahalme <jarno@covalent.io>

jrajahalme committed:
Docs: Troubleshooting updates.

Minor fixes to the troubleshooting docs.

Signed-off-by: Jarno Rajahalme <jarno@covalent.io>
